### PR TITLE
Fixed search bar issue on Chrome and Safari

### PR DESCRIPTION
**Pull request by dotjamie**
Referencing https://github.com/Dogfalo/materialize/issues/3161 

This fixes the bug in Chrome and Safari (previous commit https://github.com/Dogfalo/materialize/commit/c129dc17f76b799d7dc68a550db04c8a53bcdae1 solves only Chrome issue)

### CODE REVIEW

File: sass/components/_navbar.scss [hunk 1]
```diff
@@ -21,6 +21,12 @@ nav {
     position: relative;
     height: 100%;
   }
+    form {
```
Comment (gugerard) [inline sass/components/_navbar.scss]: Indentation ?

**Comment by gugerard** [comment]
Thanks, it works well for me.

**Comment by keyscores** [comment]
+1 can we merge this?

**Comment by gugerard** [comment]
Actually this issue might be duplicated with : https://github.com/Dogfalo/materialize/issues/3161 which is already merged (more code is changed as part of this PR though).

**Comment by dotjamie** [comment]
@gugerard The original issue mentions a cursor bug in Safari and this addresses that as well as the Chrome fix

**Comment by acburst** [comment]
There is duplicated code in this pull request. Take a look at the whole file, your form and input-field styles are duplicates

**Pull request closed without merging**